net.hovancik.Stretchly.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: net.hovancik.Stretchly
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+separate-locales: false
+finish-args:
+  - --device=dri
+  - --env=GIO_EXTRA_MODULES=/app/lib/gio/modules/
+  - --share=ipc
+  - --socket=x11
+  # todo While Stretchly can be run under Wayland, there are still some issues that need to be ironed out.
+  # - --socket=fallback-x11
+  # - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: stretchly
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/node14/bin
+      env:
+        XDG_CACHE_HOME: /run/build/stretchly/flatpak-node/cache
+        npm_config_cache: /run/build/stretchly/flatpak-node/npm-cache
+        npm_config_nodedir: /usr/lib/sdk/node14
+        npm_config_offline: 'true'
+    subdir: main
+    sources:
+      - type: dir
+        path: .
```

Comment:
not sure what you are trying to do here, but surely this doesn't look right at all.

Reply:
this doesn't do what you think it does. `mkdir -p` can be used to create the directory.

Reply:
I can't get this to work using `mkdir -p`. I've tried both of the following and get the same error above.

```yaml
- mkdir -p main
- npm install --offline
```

```yaml
- mkdir -p "$FLATPAK_BUILDER_BUILDDIR/main"
- npm install --offline
```

Reply:
I figured this out. Fixed.